polkadot/primitives/src/v8/mod.rs

```diff
@@ -426,7 +426,7 @@ pub const MAX_HEAD_DATA_SIZE: u32 = 1 * 1024 * 1024;
 /// * checking updates to this stored runtime configuration do not exceed this limit
 /// * when detecting a PoV decompression bomb in the client
 // NOTE: This value is used in the runtime so be careful when changing it.
-pub const MAX_POV_SIZE: u32 = 5 * 1024 * 1024;
+pub const MAX_POV_SIZE: u32 = 10 * 1024 * 1024;
```

Comment:
Did we validate this works e2e, like have a parachain that generates 10MiB PoV and confirm parachain blocks works at 6s rate with all blocks included on chain ?

Reply:
It kind of makes sense to add a glutton test with these 10MB PoVs. Later we should do a more scaled up test on Versi.

Reply:
We haven't validated anything yet (beyond running local zombienets to ensure 10 Mb PoVs work in principle). The purpose of this PR is to gather opinions and concerns, we're not merging it before we do a lot of tests anyway.

Reply:
> beyond running local zombienets to ensure 10 Mb PoVs work in principle

That's what I was asking for :D.

Reply:
> The purpose of this PR is to gather opinions and concerns, we're not merging it before we do a lot of tests anyway.

But the description of the PRs and my own thing tells me that we can merge it. PoV should still be limited by the relay runtime config value at 5MB. To have 10MB in practice you need governance to bump the chain limit and also parachain runtimes to be adjusted.

Reply:
Well, okay, I'll put that another way. By this PR, I argue that the given changeset is enough to enable 10 Mb PoVs, and I expect others to argue why it is not. @alexggh's point is that we need more testing before applying the changes, and that's fair. By now, I've finished the local tests with gluttons and will do some stress tests. After that, I'm ready to go to Versi, but I'd like to make it as close to real-life as possible on Versi, that is, to start with a regular network with 5 Mb PoVs and then upgrade it to 10 Mb step by step.

But if the lack of testing is the only argument against merging it in this form, it's beautiful ;)

Reply:
> @alexggh's point is that we need more testing before applying the changes, and that's fair.

I was simply asking, because I was curious what tests we've done on this end, the fact that you ran it locally with 10MiB is good enough for me. 

I understand that the relay-chain configuration is guarding us against using more than 5MiB, and it is probably safe to merge it without positive evidence that the new  maximum is working all the way trough our stack, I was simply trying to avoid the situation where we raise this constant because we know the relay-chain configuration is at 5MiB and it is risk free now, but then 6 months later someone else comes without this context and raises the relay-chain configuration to 10MiB because  MAX_POV_SIZE is already 10MiB and it must be safe :D.